### ISSUE TITLE
ACS setup help on FOSS build (app ops)

### ISSUE THREAD

**Issue by MarkyMarkDE** [d4rken-org/sdmaid-se#728]
If you are able to detect to which option you have to go after installing or update SDMaid SE on Android, following would be nice:

When you install or update SDMaid SE FOSS and open SDMaid SE UI, you will asked for activation of the assessibility mode and you click on the button in the UI, then you will be "directed" on the main setting for the accessibility mode but when you install new or update, you have to go to apps and click on SDMaid SE and on the upper right clicking on the tree dots and enable the "advanced" setting for SDMaid SE.

It would be nice and always useful if you are able to detect which option you must redirect, do it's more comfortable for a "normal" user.

**Comment by d4rken** [comment]
This is already implemented, is it not working?

![Screenshot from 2024-01-25 13-05-49](https://github.com/d4rken-org/sdmaid-se/assets/1439229/e9a5f317-2493-40df-8491-54b532536b65)

There is no official API for this, so it might not work in every case but I think it's as good as it gets:

https://github.com/d4rken-org/sdmaid-se/blob/ecd8f2c18cd6a48ae9bd64c8bd384cc2519f8daa/app/src/main/java/eu/darken/sdmse/setup/automation/AutomationSetupModule.kt#L56-L78

The blue extra hint about the restrictions shows if the following conditions are met:

* The accessibility service was never launched before
* The user has tried to enable the accessibility service before (we don't know that, but just assume that the user tried it if they opened the system screen via SD Maid). The user has to have tried this first, otherwise the extra option in the app settings is not shown by the system :smiling_face_with_tear:.
* The accessibility service might be restricted (has the app been sideloaded, not gplay, not fdroid)
* SD Maid has neither root nor shizuku, so the service can't be self-enabled.

**Comment by MarkyMarkDE** [comment]
no, if you install sd-maid, and you click on e.g. "settings" or "grant access" for acessibility service or [...] you get the information that no app is found for this action

**Comment by d4rken** [comment]
> no, if you install sd-maid, and you click on e.g. "settings" or "grant access" for acessibility service or [...] you get the information that no app is found for this action

On what device was this?

**Comment by MarkyMarkDE** [comment]
> > no, if you install sd-maid, and you click on e.g. "settings" or "grant access" for acessibility service or [...] you get the information that no app is found for this action
> 
> On what device was this?

this was on waipu TV Stick with AndroidTV 13, not sure, if this also happens on waipu TV Box with AndroidTV 14 ...
